client/components/forms/form-phone-input/test/index.jsx

```diff
@@ -1,77 +1,57 @@
 /**
  * External dependencies
  */
-var expect = require( 'chai' ).expect,
-	useFakeDom = require( 'test/helpers/use-fake-dom' ).withContainer,
-	getContainer = require( 'test/helpers/use-fake-dom' ).getContainer,
-	useMockery = require( 'test/helpers/use-mockery' );
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { expect } from 'chai';
+import { withContainer as useFakeDom } from 'test/helpers/use-fake-dom';
 
-describe( 'FormPhoneInput', function() {
-	var React, ReactDom, ReactClass, TestUtils, i18n, mockCountriesList, mockCountriesListEmpty, FormPhoneInput, countries, container;
-
-	useFakeDom();
-	useMockery();
-
-	before( function() {
-		ReactDom = require( 'react-dom' );
-		React = require( 'react' );
-		ReactClass = require( 'react/lib/ReactClass' );
-		TestUtils = require( 'react-addons-test-utils' );
-
-		i18n = require( 'i18n-calypso' );
-		mockCountriesList = require( './mocks/mock-countries-list' );
-		mockCountriesListEmpty = require( './mocks/mock-countries-list-empty' );
-
-		countries = mockCountriesList.get();
+/**
+ * Internal dependencies
+ */
+import { FormPhoneInput } from '../';
+import mockCountriesList from './mocks/mock-countries-list';
+import mockCountriesListEmpty from './mocks/mock-countries-list-empty';
 
-		ReactClass.injection.injectMixin( i18n.mixin );
-		FormPhoneInput = require( 'components/forms/form-phone-input' );
-		container = getContainer();
-	} );
+const countries = mockCountriesList.get();
 
-	afterEach( function() {
-		ReactDom.unmountComponentAtNode( container );
-	} );
+describe( 'FormPhoneInput', function() {
+	useFakeDom();
 
 	describe( 'getValue()', function() {
 		it( 'should set country from props', function() {
-			var phoneComponent = ReactDom.render( <FormPhoneInput countriesList={ mockCountriesList } initialCountryCode={ countries[ 1 ].code } />, container );
-
-			expect( phoneComponent.getValue().countryData ).to.deep.equal( countries[ 1 ] );
+			const phoneComponent = shallow( <FormPhoneInput countriesList={ mockCountriesList } initialCountryCode={ countries[ 1 ].code } /> );
```

Comment:
Let's split those props on separate lines to avoid the ESLint line length warning.